python-lib/utils/dkulib_io_utils.py

```diff
@@ -36,16 +36,4 @@ def set_column_descriptions(
         )
```

Comment:
Same remark as in other PR:
No need for the `iter([...])` , you can directly use `next(... for ... in ... if ..., None)`

Reply:
Actually it doesn't work, raises th following error : 
`Error in Python process: At line 3: <class 'SyntaxError'>: Generator expression must be parenthesized if not sole argument`